File: app/src/main/java/com/zhaoweihao/mrtranslator/view/MyItemDecoration.java
package com.zhaoweihao.mrtranslator.view;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by Zhaoweihao on 17/7/6.
 */

public class MyItemDecoration extends RecyclerView.ItemDecoration{

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        outRect.set(0, 0, 0, 1);
    }
}

File: app/src/main/java/com/zhaoweihao/mrtranslator/view/MyItemDecoration.kt
package com.zhaoweihao.mrtranslator.view

import android.graphics.Rect
import android.support.v7.widget.RecyclerView
import android.support.v7.widget.RecyclerView.ItemDecoration
import android.view.View

/**
 * Created by Zhaoweihao on 17/7/6.
 */
class MyItemDecoration : ItemDecoration() {
    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect[0, 0, 0] = 1
    }
}